src/entrypoint/graphql.ts

```diff
@@ -244,7 +244,7 @@ const resolvers = {
 const isAuthenticated = rule({ cache: 'contextual' })(
   async (parent, args, ctx, info) => {
     if(ctx.uid === null) {
-      return new Error(`${util.inspect({ message: 'Not authorised!', request: ctx.request.body }, false, Infinity)}`)
+      throw new AuthorizationError(undefined, {logger: graphqlLogger, request: ctx.request.body})
```

Comment:
oh we were returning an error which doesn't make sense.
can you verify the throw error is handled correctly? (ie: it doesn't make graphql restart?)

Reply:
Yep, already did that. Works as expected.